Coffee/Coffee/Views/MenuDetail/MenuDetailVC.swift

```diff
@@ -0,0 +1,445 @@
+        txtView.textColor = UIColor.lightGray
+        txtView.text = placeHolder
+        lblCount.text = String(count)
+    }
+    
+    private func setupBindings(){
+        [btnSmall,btnMedium,btnLarge,btnHot,btnCold,btnNone,btn30,btn50].addTarget(selector: #selector(onChangeSelection(_:)))
+        
+        txtMilkpick.addTarget(self, action: #selector(onMilkChange), for: .editingDidBegin)
+        txtToppingpick.addTarget(self, action: #selector(onToppingChange), for: .editingDidBegin)
+        
+        btnMinus.addTarget(self, action: #selector(decrementTapped), for: .touchUpInside)
+        btnPlus.addTarget(self, action: #selector(incrementTapped), for: .touchUpInside)
+        
+        btnMilk.addTarget(self, action: #selector(showMilkPicker), for: .touchUpInside)
+        btnTopping.addTarget(self, action: #selector(showToppingPicker), for: .touchUpInside)
+        
+        pickerView.delegate = self
+        pickerView.dataSource = self
+        
+        txtMilkpick.delegate = self
+        txtToppingpick.delegate = self
+        
+        txtView.delegate = self
+        
+    }
+    
+    @objc func onMilkChange(){
+        showPicker(for: .MILK, textField: txtMilkpick)
+    }
+    
+    @objc func onToppingChange(){
+        showPicker(for: .TOPPING, textField: txtToppingpick)
+    }
+    
+    func showPicker(for pickerType: PickerType, textField: UITextField) {
+        currentPickerType = pickerType
+        textField.inputView = pickerView
+        textField.inputAccessoryView = getToolbar()
+        pickerView.reloadAllComponents()
+        textField.becomeFirstResponder()
+    }
+
+    @objc func showMilkPicker() {
+        showPicker(for: .MILK, textField: txtMilkpick)
+    }
+
+    @objc func showToppingPicker() {
+        showPicker(for: .TOPPING, textField: txtToppingpick)
+    }
+    
+    @objc func doneBtnTapped(){
+        switch currentPickerType {
+        case .MILK:
+            if (txtMilkpick.text == "" || ((txtMilkpick.text?.isEmpty) != nil)){
+                txtMilkpick.text = milkOptions?[0]
+            }
+            txtMilkpick.resignFirstResponder()
+        case .TOPPING:
+            if (txtToppingpick.text == "" || ((txtToppingpick.text?.isEmpty) != nil)){
+                txtToppingpick.text = toppingOptions?[0]
+            }
+            txtToppingpick.resignFirstResponder()
+        default:
+            print()
+        }
+    }
+    
+    
+    func applyShadow(to view: UIView) {
+        view.layer.shadowColor = UIColor.black.cgColor // Set shadow color
+        view.layer.shadowOpacity = 0.3 // Set shadow opacity (0.0 to 1.0)
+        view.layer.shadowOffset = CGSize(width: 0, height: 4) // Set shadow offset
+        view.layer.shadowRadius = 5 // Set shadow blur radius
+        view.layer.masksToBounds = false // Disable clipping to show shadow
+        view.layer.cornerRadius = 10
+    }
+    
+    func initialState(){
+        //For Size
+        updateSelection(
```

Comment:
I think we can be done this UI using only Button which already have normal and selected states. It will help no need to resign normal state and selected state and just update `isSelected`.
Or use collection view to be more dynamic.

plz try using button only. And can create separate new function to update background color and font for selected state and normal state.

Reply:
Hok sayr.. I tried with button but I can't control the Icon image and UILabel inside the UIButton. That's why I used UIView and StackView and cover with UIButton over that UIView. 
Could you check me on Tomorrow night sayar?  I also resolved the conflict too. Please check sayar.
I will upate my UI with Font and also trying with UIButton only sayar.
Thank you for your review .

Reply:
can use `UIButton..titleLabel?.font` for label.
use icon with exact same size

try to check UIButton.Configuration:

Reply:
hok sayar. i will try again